### ISSUE TITLE
Actions don't seem to appear anymore on attributes tab

### ISSUE THREAD

**Issue by frater** [sdenec/tidy5e-sheet#675]
Not sure if this is a purposeful change that i've missed or something wrong with my settings, but the sheet has become pretty much unusable for my PCs.  The NPC sheets are back working again after the latest fixes, but i'm still not seeing what I expected on the PC sheets.  

Previously the main page of the sheet, attributes, had a short list of the common features for a character - weapons, actions etc.  These still display for NPC sheets, but no longer for PC.

![image](https://user-images.githubusercontent.com/2861963/216545937-6705d401-f399-4f97-b79b-96f7490e79e0.png)

This is an NPC sheet for a familiar, showing the standard actions.

![image](https://user-images.githubusercontent.com/2861963/216546236-216c7d5e-14f7-46a9-b8d8-bb7a6c4bb2e5.png)

This is what we see for the PC sheets, it's simply missing that section.

I've tried uninstalling and reinstalling, running through the config options, resetting options to default, and digging through the code and console.  No errors are showing up in the console to give any ideas, and though the code is different between NPC sheet and PC Sheet (with the npc sheet using a seperate template module for this and the tidy5e sheet looking like it has a div with an actions-target that I can't find ever used), that doesn't look to be something that's changed recently.

**Comment by p4535992** [comment]
Did you mean the favorite section of the Charater Sheet ? If you do not set something as favorite you don't see anything.

**Comment by frater** [comment]
That section for our pc's, previously, showed the same sort of feature display as the NPC sheet does there.

It's slightly different to just setting favorites (which we've never done) as it seperates out attacks, features etc.)

**Comment by sdenec** [comment]
There never was any other section in the empty space below resources on PC sheets than the favorites. Maybe you are missing another module?

**Comment by frater** [comment]
This is really strange.  We haven't updated anything except foundry itself (to v10 which broke tidy earlier this year).  Prior to then, we've been using it for 12 months and that attributestab always showed the same as the npc sheet does on the abilities tab.  No idea why, if it wasn't meant to.

Oh well, please close this as some weirdness on my end, we'll try playing around with favorites and see how it goes.  Thanks

**Comment by p4535992** [comment]
The only module with this feature i have found is [Character Actions](https://github.com/ElfFriend-DnD/foundryvtt-dnd5eCharacterActions) but i see on the screenshot you already used that

**Comment by frater** [comment]
Yeah, I had never used that module before today, trying to sort out the issue before our session started (we're in game now :P)

**Comment by DivertedCircle** [comment]
Abilities/Features that were set as Favorites got reset with the Tidy5e update. You need to again reflag abilities as Favorites.

**Comment by p4535992** [comment]
@DivertedCircle that is a different issue and you must open another ticket about, but i cannot replicate the issue , have you tried to disable all others modules ? or reset the module settings ?

**Comment by DivertedCircle** [comment]
> @DivertedCircle that is a different issue and you must open another ticket about it, but i cannot replicate the issue , have you tried to disable all others modules ? or reset the module settings ?

It was not a reoccurring issue. It happened a single time after the recent 5e system updates and since then we've been able to reassign abilities as favorites with no issues. I believe this is what the original poster is seeing, but they missed that they just need to reflag abilities as favorites again.

**Comment by BillPitcher** [comment]
Same, did the tidy5e-sheet update and all the players Favourites disappeared. Players had to scramble pre game to redo them.

**Comment by p4535992** [comment]
I have analyzed the problem and without going into too much technical detail, I have more disadvantages than advantages in fixing the backward compatibility of the favorites, so to force everyone to use the new data, the issue will not be solved, you will have to waste 10 minutes setting the favorites.

**Comment by eltariel** [comment]
My group also noticed the actions list disappearing from the attributes page. It's one of the main reasons we're not using this sheet for PCs anymore. We never used the favourites feature because there's no indication that it even exists beyond the toggle buttons on the item lists.

> There never was any other section in the empty space below resources on PC sheets than the favorites. Maybe you are missing another module?

Are you sure?

The placeholder for the actions list was added in 0de2a89 along with code to render the actions list to it. I manually updated `scripts/tidy5e-sheet.js` to do that rather than create the tab and I got this when I opened the sheet: 
![image](https://user-images.githubusercontent.com/12076881/224604456-e47706f1-d751-4739-a839-cbcd88c1b8f2.png)

It looks ok if I stretch the window even though it's clearly styled to be a standalone tab... 
![image](https://user-images.githubusercontent.com/12076881/224604741-27f9aeff-46e0-4c9c-83ea-8023db244348.png)

Mind if I clean this up and submit a PR?
